packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -374,6 +374,8 @@ export abstract class Chart extends Observable {
             seriesRegion.addListener('contextmenu', (event) => this.onContextMenu(event), All),
             ctx.keyNavManager.addListener('browserfocus', (event) => this.onBrowserFocus(event)),
             ctx.interactionManager.addListener('page-left', () => this.destroy()),
+            ctx.interactionManager.addListener('enter', () => this.update(ChartUpdateType.SCENE_RENDER)),
+            ctx.interactionManager.addListener('leave', (event) => this.onLeave(event, true)),
```

Comment:
This might not be necessary - I think this might be able to be handled in the regionManager, but I wasn't too sure

Reply:
These do look a bit redundant to my eye - can you try removing these and seeing if things behave as expected?

Reply:
It definitely didn't work without this - but I suspect it's possible to handle this somewhere better